go/store/util/outputpager/page_output.go

```diff
@@ -110,3 +137,7 @@ func RegisterOutputpagerFlags(flags *flag.FlagSet) {
 func IsStdoutTty() bool {
 	return goisatty.IsTerminal(os.Stdout.Fd())
 }
+
+func SetTestingArg(s bool) {
```

Comment:
Doesn't need to be exported

Reply:
it's used in `TestLogSigTerm` in log_test.go